srcpkgs/itch/files/itch

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+itch-setup --prefer-launch -- "@$"
```

Comment:
Maybe ask for upstream to include the script? It looks useful.

Reply:
Also, it should be `exec itch-setup`

Reply:
This code is provided by the project, I am following the recommendation, the project does not have a script itself, but it does provide this code.

Reply:
In this case we know better; use `exec`.

Reply:
With EXEC, it occurs that application understands, each new startup is the first, due there is it, always does the framework download, even if you have done it previously!

2021/07/19 22:29:59 itch-setup will log to /tmp/itch-setup-log.txt
2021/07/19 22:29:59 =========================================
2021/07/19 22:29:59 itch-setup "head, no build date" starting up at "2021-07-19 22:29:59.997866826 -0300 -03 m=+0.023844080" with arguments:
2021/07/19 22:29:59 "itch-setup"
2021/07/19 22:29:59 =========================================
2021/07/19 22:29:59 App name detected: itch
2021/07/19 22:29:59 Locale:  en-US
2021/07/19 22:30:00 Initializing installer GUI...
2021/07/19 22:30:00 Using GTK UI
2021/07/19 22:30:00 Initializing (itch) multiverse @ (/home/bugcrazy/.itch)
2021/07/19 22:30:00 (/home/bugcrazy/.itch)(current = "25.5.1", ready = "")
2021/07/19 22:30:00 Loading image resources...
2021/07/19 22:30:00 Setting up progress bar...
2021/07/19 22:30:00 Positioning and showing window...
2021/07/19 22:30:01 Will install version 25.5.1
2021/07/19 22:30:03 Reading signature...
2021/07/19 22:30:03 Installing 197.43 MiB (295 files, 25 dirs, 0 symlinks)
2021/07/19 22:30:03 Looks like (25.5.1) is already installed to (/home/bugcrazy/.itch/app-25.5.1)
2021/07/19 22:30:03 Let's just heal that
2021/07/19 22:30:03 Installing to (/home/bugcrazy/.itch/app-25.5.1)
2021/07/19 22:30:03 Healing (/home/bugcrazy/.itch/app-25.5.1)...
2021/07/19 22:30:07 0 B was healed @ 0 B/s (4s total)
2021/07/19 22:30:07 Healed in-place
2021/07/19 22:30:07 No validate handler, assuming good!
2021/07/19 22:30:07 Determining whether or not we've been installed via an OS package...
2021/07/19 22:30:07 Our execPath (/usr/bin/itch-setup) is somewhere in /usr, not installing desktop files
2021/07/19 22:30:07 Launching (25.5.1) from (/home/bugcrazy/.itch/app-25.5.1)
2021/07/19 22:30:07 Kernel should support SUID sandboxing, leaving it enabled
2021/07/19 22:30:07 App launched, getting out of the way

With the standard argument this does not occur. 

2021/07/19 22:36:37 itch-setup will log to /tmp/itch-setup-log.txt
2021/07/19 22:36:37 =========================================
2021/07/19 22:36:37 itch-setup "head, no build date" starting up at "2021-07-19 22:36:37.731537432 -0300 -03 m=+0.078722338" with arguments:
2021/07/19 22:36:37 "itch-setup"
2021/07/19 22:36:37 "--prefer-launch"
2021/07/19 22:36:37 "--"
2021/07/19 22:36:37 "@$"
2021/07/19 22:36:37 =========================================
2021/07/19 22:36:37 App name detected: itch
2021/07/19 22:36:37 Locale:  en-US
2021/07/19 22:36:37 Initializing installer GUI...
2021/07/19 22:36:37 Using GTK UI
2021/07/19 22:36:37 Initializing (itch) multiverse @ (/home/bugcrazy/.itch)
2021/07/19 22:36:37 (/home/bugcrazy/.itch)(current = "25.5.1", ready = "")
2021/07/19 22:36:37 Launch preferred, attempting...
2021/07/19 22:36:37 Launching (25.5.1) from (/home/bugcrazy/.itch/app-25.5.1)
2021/07/19 22:36:37 Kernel should support SUID sandboxing, leaving it enabled
2021/07/19 22:36:37 App launched, getting out of the way

Reply:
Huh, why would that happen...

Reply:
This is the author's thing of the software, who preferred to use this arguments.